packages/rancher-operator-crd/package.yaml

```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3-rc3/rancher-operator-crd-0.1.3-rc3.tgz
+url: https://github.com/rancher/rancher-operator/releases/download/v0.1.3-rc4/rancher-operator-crd-0.1.3-rc4.tgz
 packageVersion: 00
-releaseCandidateVersion: 03
```

Comment:
We should remove this field `releaseCandidateVersion`

Reply:
remove the field or not update it? I'm using https://github.com/rancher/charts/pull/967/files as an example

Reply:
nvm